### PR TITLE
Prod landing pages

### PR DESCRIPTION
**Pull request by suehighmoor**
Added landing pages for 5 main cloud stacks

### CODE REVIEW

File: articles/azure/azs-home.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-create-vm-terraform.html">Create a VM (using Terraform)</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-configure-powershell-users.html">Configure PowerShell environment</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/azure/azs-how-use-azure-api.html">Use the Azure Stack API</a></ul></div>
+</div>
+
+## Azure Stack portal overview
+
+<div class="row">
+  <div class="col-md-6">
+    <div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/305064724?color=ffffff&title=0&byline=0&portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
```
Comment (ollie-nye) [inline articles/azure/azs-home.md]: Could the styles in here be extracted out to the CSS file, keep it a bit tidier
Could newlines be added between elements too to aid readability

File: articles/openstack/ostack-home.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/openstack/ostack-how-create-heat-template.html">Create and deploy a Heat template</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/openstack/ostack-how-create-load-balancer.html">Create load balancing services</a></ul></div>
+  <div class="col-md-3"><ul><li><a href="https://docs.ukcloud.com/articles/openstack/ostack-how-recover-instance.html">Recover from failure</a></ul></div>
+</div>
+
+## Useful Videos
+
+<div class="row">
+  <div class="col-md-6">
+    <div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/305934797?color=ffffff&title=0&byline=0&portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
```
Comment (ollie-nye) [inline articles/openstack/ostack-home.md]: Same comment here about styles/newlines between elements

File: articles/azure/azs-home.md (hunk 1 above)
Comment (suehighmoor) [reply articles/azure/azs-home.md]: Added line breaks

File: articles/openstack/ostack-home.md (hunk 2 above)
Comment (suehighmoor) [reply articles/openstack/ostack-home.md]: Added line breaks

**Comment by suehighmoor** [comment]
@cblackuk, @ollie-nye, if you're happy with these pages as v1, could you go ahead and approve please.

**Pull request merged**